README.md

```diff
@@ -328,6 +328,256 @@ npm install prosemirror-utils
+       |      |      |             |
+    0  |  A1  |  B1  |     C1      |
+       |______|______|______ ______|
+       |      |             |      |
+    1  |  A2  |     B2      |      |
+       |______|______ ______|      |
+       |      |      |      |  D1  |
+    2  |  A3  |  B3  |  C2  |      |
+       |______|______|______|______|
+       |      |             |      |
+    3  |  A4  |     B4      |      |
+       |______|______ ______|      |
+       |      |      |      |  D2  |
+    4  |  A5  |  B5  |  C3  |      |
+       |______|______|______|______|
+
+
+   If you try to move the row 0 to position 4 with "tryToFit" enabled, by default,
+   the code will put it on after the merged rows,
+   but you can override it using the "direction" option.
+
+   -1: Always put the origin before the target
+        ____________________________
+       |      |             |      |
+    0  |  A2  |     B2      |      |
+       |______|______ ______|      |
+       |      |      |      |  D1  |
+    1  |  A3  |  B3  |  C2  |      |
+       |______|______|______|______|
+       |      |      |             |
+    2  |  A1  |  B1  |     C1      |
+       |______|______|______ ______|
+       |      |             |      |
+    3  |  A4  |     B4      |      |
+       |______|______ ______|      |
+       |      |      |      |  D2  |
+    4  |  A5  |  B5  |  C3  |      |
+       |______|______|______|______|
+
+    0: Automatically decide the best place to fit
+        ____________________________
+       |      |             |      |
+    0  |  A2  |     B2      |      |
+       |______|______ ______|      |
+       |      |      |      |  D1  |
+    1  |  A3  |  B3  |  C2  |      |
+       |______|______|______|______|
+       |      |             |      |
+    2  |  A4  |     B4      |      |
+       |______|______ ______|      |
+       |      |      |      |  D2  |
+    3  |  A5  |  B5  |  C3  |      |
+       |______|______|______|______|
+       |      |      |             |
+    4  |  A1  |  B1  |     C1      |
+       |______|______|______ ______|
+
+    1: Always put the origin after the target
+        ____________________________
+       |      |             |      |
+    0  |  A2  |     B2      |      |
+       |______|______ ______|      |
+       |      |      |      |  D1  |
+    1  |  A3  |  B3  |  C2  |      |
+       |______|______|______|______|
+       |      |             |      |
+    2  |  A4  |     B4      |      |
+       |______|______ ______|      |
+       |      |      |      |  D2  |
+    3  |  A5  |  B5  |  C3  |      |
+       |______|______|______|______|
+       |      |      |             |
+    4  |  A1  |  B1  |     C1      |
+       |______|______|______ ______|
+
+   ```javascript
+   dispatch(
+     moveRow(x, y, options)(state.tr)
+   );
+   ```
+
+
+ * **`moveColumn`**`(originColumnIndex: number, targetColumnIndex: targetColumnIndex, options: ?MovementOptions) → fn(tr: Transaction) → Transaction`\
```

Comment:
see my comments for `moveRow`, the same goes for `moveColumn`

Reply:
Hmm, same thing here too.